Percy/utils/utils.cs

```diff
@@ -0,0 +1,120 @@
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+
+namespace PercyIO.Appium
+{
+  internal class Utils
```

Comment:
```suggestion
  internal class RefectionUtils
```
Move non reflection methods if any outside to a separate Util class

Reply:
done